processor/integrations/integrations.go

```diff
@@ -181,7 +183,7 @@ func GetDestinationIDs(clientEvent interface{}, destNameIDMap map[string]backend
 
 //GetDestinationURL returns node URL
 func GetDestinationURL(destID string) string {
-	return fmt.Sprintf("%s/v0/%s", destTransformURL, strings.ToLower(destID))
+	return fmt.Sprintf("%s/v0/%s?whSchemaVersion=%s", destTransformURL, strings.ToLower(destID), validators.GetWHSchemaVersion())
```

Comment:
Instead of the dependency between validators and integrations. Can we do the following.

```
---main.go
	if !validators.ValidateEnv() {
		panic(errors.New("Failed to start rudder-server"))
	}

    integrations.SetWarehouseSchemaVersion(validators.GetWHSchemaVersion())

```

Reply:
moved to config